docs/blog/2019-08-06-gazette-august/index.md

```diff
@@ -0,0 +1,125 @@
+
+### WPGraphQL for Advanced Custom Fields
+
+[WPGraphQL for Advanced Custom Fields](https://www.wpgraphql.com/acf/) is now open source. [Jason Bahl](https://twitter.com/jasonbahl) recently joined our team and open sourcing this is a step forward in our commitment to making Gatsby + WordPress seamless.
+
+### Learning
+
+Our learning team has been hard at work on improving the documentation on [gatsbyjs.org](https://gatsbyjs.org)
+
+Amongst many many other things, we've added
+
+- Next and previous pagination links to guide users through the documentation
+- A floating Table of Contents on most pages
+
+[Marcy](https://twitter.com/marcysutton) also taught a workshop at Frontend Masters and this is a must watch for all of us to learn how to build accessible applications on the web.
+
+Check out the workshop at https://frontendmasters.com/workshops/javascript-accessibility/
+
+### Jason _Blitz_ Lengstorf
+
+Jason has basically been living his life on livestream since the past month. He's done some incredible streams and has many more exciting ones coming up.
+
+My favourite of the past few ones was [Advanced GraphQL techniques in Gatsby](https://www.twitch.tv/videos/462874512) with [Mikhail Novikov](https://twitter.com/freiksenet).
+
+In case that isn't your jam, there's also [Gatsby + WordPress](https://www.youtube.com/watch?v=DH7I1xRrbxs), [building an RSS feed-powered podcast site](https://www.youtube.com/watch?v=0hGlvyuQiKQ), [composing and styling Gatsby themes](https://www.youtube.com/watch?v=6Z4p-qjnKCQ), [Microfrontends](https://www.youtube.com/watch?v=0Ta-awtLZTs) and even a [livestream on livestreaming](https://www.youtube.com/watch?v=rgTugjTDYaE)!
+
+Here's a convenient [playlist](https://www.youtube.com/playlist?list=PLz8Iz-Fnk_eTpvd49Sa77NiF8Uqq5Iykx) if you want to binge watch these.
+
+### Theme Jam
+
+Talking about jam, to celebrate the launch of Gatsby Themes, we announced the [Gatsby Theme Jam](https://themejam.gatsbyjs.org/) last month.
+
+The response was incredible. Seeing so many wonderful themes from all of you has been spectacular and we're afraid we might like _too_ many of them to pick one.
+
+The team is currently hard at work judging and picking their favourites so results should be out very soon!
+
+## Community Highlights
+
+Firstly, thank you for all your contributions, folks! Every single contribution to Gatsby (however small or big) makes it better. Some of the community highlights from the last month include:
+
+- [lannonbr](https://github.com/lannonbr) [built a Table of Contents component](https://github.com/gatsbyjs/gatsby/pull/15251) that is live on our documentation and is great when reading a lengthy documentation page
```

Comment:
Comment/nit should we use names here?

Reply:
Good idea! Handles work in GitHub land but names are better on the blog post for sure. Done.